css/main.css

```diff
@@ -296,7 +296,44 @@ nav {
   }
 }
 
-@media (max-width: 1100px) {
+@media (max-width: 870px) {
+  nav {
+    display: none;
+  }
+  .nav-show {
+    display: block;
+  }
+
+  .nav-option-bar {
+    background: #F8AC45;
+    padding: 0;
+    position: fixed;
+    right: -400px;
+    width: 400px;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    -webkit-transition: .25s ease-in-out;
+    -moz-transition: .25s ease-in-out;
+    -o-transition: .25s ease-in-out;
+    transition: .25s ease-in-out;
+  }
+
+  .nav-option {
+    border-bottom: dotted 2px;
```

Comment:
You should include a color here, as otherwise the border will assume currentColor (the current value of the "color" property).